nexmo/__init__.py

```diff
@@ -16,7 +16,7 @@
 
 if sys.version_info[0] == 3:
     string_types = (str, bytes)
-    from urllib.parse import urlparse
+    from urllib.parse import urlparse # coming up with error "no name 'parse' in module 'urllib'"
```

Comment:
You're seeing this because you're running Python 2 code analysis. Python 3 __does__ have a `urllib.parse` module, and this if block is only executed under Python 3.